src/main/java/blockrenderer6343/client/renderer/WorldSceneRenderer.java

```diff
@@ -242,39 +242,44 @@ protected void drawWorld() {
         glEnable(GL_TEXTURE_2D);
         glEnable(GL_ALPHA_TEST);
 
+        final int savedAo = mc.gameSettings.ambientOcclusion;
+        mc.gameSettings.ambientOcclusion = 0;
+
         Tessellator.instance.startDrawingQuads();
-        Tessellator.instance.setBrightness(15 << 20 | 15 << 4);
-        for (BlockPosition pos : renderedBlocks) {
-            Block block = world.getBlock(pos.x, pos.y, pos.z);
-            if (block.equals(Blocks.air)) continue;
-            block.setLightLevel(15);
-
-            RenderBlocks bufferBuilder = new RenderBlocks();
-            bufferBuilder.blockAccess = world;
-            bufferBuilder.setRenderBounds(0, 0, 0, 1, 1, 1);
-            bufferBuilder.renderAllFaces = renderAllFaces;
-            if (block instanceof BW_GlasBlocks) {
-                // this mod cannot render renderpass = 1 blocks for now
-                bufferBuilder.renderStandardBlockWithColorMultiplier(
-                        block,
-                        pos.x,
-                        pos.y,
-                        pos.z,
-                        ((BW_GlasBlocks) block).getColor(world.getBlockMetadata(pos.x, pos.y, pos.z))[0] / 255f,
-                        ((BW_GlasBlocks) block).getColor(world.getBlockMetadata(pos.x, pos.y, pos.z))[1] / 255f,
-                        ((BW_GlasBlocks) block).getColor(world.getBlockMetadata(pos.x, pos.y, pos.z))[2] / 255f);
-            } else if (!GT_Renderer_Block.INSTANCE
-                    .renderWorldBlock(world, pos.x, pos.y, pos.z, block, block.getRenderType(), bufferBuilder)) {
-                        bufferBuilder.renderBlockByRenderType(block, pos.x, pos.y, pos.z);
-                    }
-        }
-        if (onRender != null) {
-            onRender.accept(this);
+        try {
+            Tessellator.instance.setBrightness(15 << 20 | 15 << 4);
```

Comment:
I don't fully understand rendering code, but is this still needed when we override `getLightBrightnessForSkyBlocks`?

Reply:
If you don't set it I think the tesselator doesn't always add it to vertex data, so better to keep it explicitly set to be safe

Reply:
I see